### ISSUE TITLE
Maybe remove recurrent batch normalization

### ISSUE THREAD

**Issue by carlthome** [carlthome/tensorflow-convlstm-cell#6]
I'm not having much success with recurrent batch normalization so I'm thinking of removing it to simplify the code. Could anyone confirm whether recurrent batch normalization in a ConvLSTM is working for them?

**Comment by carlthome** [comment]
I like the batch normalization from Deep Speech 2 more (only on the input-to-hidden weights, nothing is normalized in the hidden-to-hidden connections) at the moment. Gladly would get some insight into pros/cons of both methods!

**Comment by carlthome** [comment]
https://github.com/carlthome/tensorflow-convlstm-cell/commit/c14d0c419b93e445ca6111923e3b0c70d5b171d9
